src/app/conf/2025/events/fost/page.tsx

```diff
@@ -0,0 +1,299 @@
+import { Metadata } from "next"
+import Image from "next-image-export-optimizer"
+
+import { Button } from "@/app/conf/_design-system/button"
+import { Accordion } from "@/app/conf/_design-system/accordion"
+
+import GraphQLFoundationWordmark from "../../assets/graphql-foundation-wordmark.svg?svgr"
+import { Hero, HeroStripes } from "../../components/hero"
+import { NavbarPlaceholder } from "../../components/navbar"
+import { CalendarIcon } from "../../../_design-system/pixelarticons/calendar-icon"
+import { PinIcon } from "../../../_design-system/pixelarticons/pin-icon"
+import { CtaCardSection } from "../../components/cta-card-section"
+
+import heroPhoto from "./hero-photo.webp"
+import { Anchor } from "../../../_design-system/anchor"
+
+export const metadata: Metadata = {
+  title: "GraphQL Day at FOST",
+}
```

Comment:
We could potentially have some JSON-LD in addition to basic metadata so it shows up in the search nicer.

https://developers.google.com/search/docs/appearance/structured-data/event